etc/skydive.yml.default

```diff
@@ -69,6 +69,11 @@ analyzer:
     # capture_gremlin: "G.V().has('Name', NE('lo'))"
     # capture_bpf: "port 80"
 
+    # By default (capture_type: "") the capture type is chosen automatically;
+    # or set here to one of pcap, afpacket, ebpf, sflow, pcapsocket, ovsmirror,
+    # dpdk, ovssflow, or ovsnetflow.
+    # capture_type: ""
```

Comment:
if capture_type is set to ""  bpf_filter/probe capabilities will not be checked 
please set to "pcap" as default and in config/config.go as well

Reply:
please not that not all capture type support BPF filtering

Reply:
Ok that's mote complicated than I thought.

Originally I had a "pcap" as default, but @safchain suggested leaving blank as default to allow skydive to choose the best capture type for each node.

My original problem was that "pcap" was hard coded for startup captures and I want to try ebpf. 

Would be glad to hear suggestions on how to allow this without confusing the user. I guess similar problems should appear in the skydive client when starting a capture.

Reply:
@nplanel It's true but this is the case for all the captures we create, not only the startup one. Setting the type as `pcap` also bring problem : if you select OVS bridges and leave `capture_type` empty, it will not work. So I think this PR has value and already fixes a problem